### DISCUSSION TITLE
The refiner "Created By"(mapped to the crawled property "Author")  showing weird refiner values

### DISCUSSION THREAD

**Discussion by SaratchandraPeddinti** [microsoft-search/pnp-modern-search#2103] [Ideas]
**Version used:**
Ex: 3.14

**Describe the bug**
The refiner "Created By"(mapped to the crawled property "Author")  showing weird refiner values

**To Reproduce**
1. Configure a "filters" webpart to show the refiners & connect it to the results webpart
2. Configure a "refinablestring" property & map it to "Author" crawled property
3. When results are returned, the refiner values are formatted in a weird way 

**Expected behavior**
Show the Author name in Last_name, First_name (as it shows in the field values)
Instead shows as:
userid@domain.com|last_name,first_name|random_number(like-43354353645646436..long one)|i:0#.f|membership|userid@domain.com

**Screenshots**
![image](https://user-images.githubusercontent.com/16175517/96739347-559d3b80-13dd-11eb-926a-d65397cb992d.png)

**Browsers:**
 - Chrome & Edge

**Comment by wobba** [comment]
If you want the creator names, you should use `ows_Author`. If you want to render like a person, use the persona template and use `ows_q_USER_Author` as the cp.

**Comment by SaratchandraPeddinti** [comment]
Hi @wobba 

I want it as name(i.e., text). In fact, I tried mapping ows_Author but it's not found in the crawled properties. Instead, there is a crawled property ows__Author (double '_' preceding author).
Then, I mapped ows_q_TEXT__Author too. No values are appearing yet.

My understanding is, we don't want the persona type with refiners & is only for results web part. Correct me if I'm wrong.

**Comment by wobba** [comment]
`ows_q_USER_Author` might be the only solution to get the name of the creator, but then add an ability in the refiner temlate to pick the display name for rendering only. This will require work on the filter web part.

**Comment by SaratchandraPeddinti** [comment]
Sure, I'll work on the code. Thanks for the response & all your work 👍

**Comment by harrychen1** [comment]
Have we completed this enhancement? I'm facing the same issue that we cannot have a filter to filter "Created by" field that is "Author" field. None of the following fields can be displayed as filter. Any suggestions? 
![image](https://user-images.githubusercontent.com/13895247/168350216-883853f2-32d9-4061-9d81-7bc2a74805d5.png)


Thanks.

**Reply by FranckyC** [reply]
@harrychen1 dit you try to  use a `RefinableStringXX` managed property mapping the `ows_Author` crawled proeprty?

**Reply by wobba** [reply]
You need to map the crawled property `Office:4` to a RefinableString to get the creator name.

**Comment by Ulee777** [comment]
Hi, 
I have similar issue. I am trying to use People:Manager property in the filter.
I got this values in the filter dropdown: 
![Error3](https://user-images.githubusercontent.com/99322930/182124402-ff7dad2c-8fb7-492a-9363-c23f48ae57f6.png)
I am using People template in the Results Webpart
I would prefer to see display name of the manager.

**Reply by wobba** [reply]
Try `{{getUserEmail <your mp>}}`. See https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#handlebars-helpers
